praw/endpoints.py

```diff
@@ -179,15 +175,14 @@
     "subreddit_about":         "r/{subreddit}/about/",
     "subreddit_filter":        "api/filter/user/{user}/f/{special}/r/{subreddit}",
     "subreddit_filter_list":   "api/filter/user/{user}/f/{special}",
-    "subreddit_random":        "r/{subreddit}/random/",
     "subreddit_settings":      "r/{subreddit}/about/edit/",
     "subreddit_stylesheet":    "r/{subreddit}/api/subreddit_stylesheet/",
     "subreddits_by_topic":     "api/subreddits_by_topic",
```

Comment:
This doesn't work anymore either.

Reply:
I put this change in the wrong commit. Fixing.

Reply:
Oh you mean subreddits_by_topic. Thanks. I'll remove that.

Reply:
Yes, sorry I should have specified. And yeah it hasn't worked for years. I discovered it didn't when writing Async PRAW but forgot to remove it.